### DISCUSSION TITLE
Media player?

### DISCUSSION THREAD

**Discussion by KingLJN** [rg35xx-cfw/rg35xx-cfw.github.io#91] [General]
Now that the ES frontend has basically working media players for background music and game videos from scraper sites (with caveats - only one audio sample rate allowed according to the doc etc), will it be possible to run the executable used by the frontend independently as an app in the PORTS/APPS section to use this as a standalone media player? Or does it need to be implemented separately?

**Comment by RazorDarkAMG** [comment]
I want it too, or Kodi integration ?

**Reply by davidpasion** [reply]
Dude wants to watch Shrek on his Anbernic

**Reply by Mookaiju** [reply]
Yes. Unironically.
